src/pretix/plugins/sendmail/models.py

```diff
@@ -227,6 +227,7 @@ class Rule(models.Model, LoggingMixin):
 
     id = models.BigAutoField(primary_key=True)
     event = models.ForeignKey(Event, on_delete=models.CASCADE, related_name='sendmail_rules')
+    subevent = models.ForeignKey(SubEvent, null=True, on_delete=models.CASCADE)
```

Comment:
`CASCADE` feels a little dangerous here, although safer than `SET_NULL`. Still, it's bad if texts just vanish, I guess, so maybe `PROTECT`? But maybe it's also annoying, I'm not sure. Any opinions from other readers?

Reply:
SET_NULL is certainly wrong, yes ;)